cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -232,17 +232,18 @@ class ReporterCommand : CliktCommand(
             }
         }
 
-        reportDurationMap.forEach { (entry, duration) ->
+        val successCount = reportDurationMap.count { (entry, duration) ->
             val (reporter, file) = entry
-            if (file.isFile) {
-                println(
+            file.isFile.also {
```

Comment:
Why do we actually need this check? Is it possible that a reporter does not throw an exception and not create the file at the same time?

Reply:
> Is it possible that a reporter does not throw an exception and not create the file at the same time?

Probably not, I'll look into changing this with a rework of this commit.

Reply:
Now I recall the reason: The "Successfully created [...]" status lines should be printed en-bloc after all reports were generated, for a better overview. And at that time the easiest way to check whether the creation of a report was successful is to check for its output file.

Reply:
Ok, but I thought if the reporter failed there would be no entry in `reportDurationMap`.

Reply:
Doh, you're right. Let me fix that.